src/main/app/src/app/zoeken/model/zoek-parameters.ts

```diff
@@ -4,14 +4,21 @@
  */
 
 export class ZoekParameters {
+
+    alleenMijnZaken: boolean = false;
+    alleenOpenstaandeZaken: boolean = false;
+    alleenAfgeslotenZaken: boolean = false;
     zoeken: { [key: string]: string } = {};
     filters: { [key: string]: string } = {};
-    filterQueries: { [key: string]: string } = {};
     datums: { [key: string]: { van: any; tot: any } } = {
         ZAAK_STARTDATUM: {
             van: null,
             tot: null
         },
+        ZAAK_EINDDATUM: {
+            van: null,
```

Comment:
Kunnen die van en tot niet sterker getypeerd worden dan any? Als het niet kan dan is het zo. Is maar een idee